environment.yml

```diff
@@ -10,6 +10,8 @@ dependencies:
   - toml=0.9.4
   - h5py=2.9.0
   - scipy=1.4.1
+  - yaml=5.3.1
```

Comment:
Should this be `pyyaml=5.3.1`? I can't see `yaml` on conda at 5.3.1 whereas 5.3.1 is the latest `pyyaml`.

Reply:
It certainly should, will fix